homework_2/program.js

```diff
@@ -0,0 +1,22 @@
+var ArgStart;
+var Test;
+var ArgSum;
+var ArgNumMax;
+var ArgNumCount;
+
+ArgStart = 2; // ignore first two arguments
+ArgSum = 0;
+ArgNumCount = 0;
+
+for (var i = ArgStart; i < process.argv.length; i++) {
+    if (isNaN(Number(process.argv[i]))) {
+       //LYN// console.log("Not a number:", (process.argv[i]))
+    } else {
+        ArgNumCount++;
+        ArgSum += Number(process.argv[i]);
```

Comment:
Since you've already converted process.argv[i] to a Number on line 12, you could probably store the result in a variable. Otherwise you're converting it from string to number twice, which might be slow if there are a lot of conversions.